internal/commands/logs/list_inputs.go

```diff
@@ -0,0 +1,83 @@
+package logs
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/10gen/realm-cli/internal/utils/flags"
+)
+
+const (
+	flagType      = "type"
+	flagTypeUsage = "specify the type(s) of logs to list, available options: []"
```

Comment:
no available options here? or does `[]` get populated by the CLI lib somehow?

Reply:
i wish, flags (and usages, etc) are something id love to find an elegant solution for at some point... but i just forgot to add these (needed to copy them from the UI first)